tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py

```diff
@@ -0,0 +1,116 @@
+import platform
+
+from envparse import env
+
+
+plt = platform.platform()
+# TODO: in integration test for PR#411. Add 'oracle-logos' and 'centos-logos'
+#  to (ol7pkgs, ol8pkgs) and (cos7pkgs, cos8pkgs) respectively
+#  as these missing packages are a result of a different cause
+#  but still missing after a rollback
+ol7pkgs = ["oracle-release-el7", "usermode", "rhn-setup", "python-syspurpose"]
+ol8pkgs = ["oracle-release-el8", "usermode", "rhn-setup", "python3-syspurpose"]
+cos7pkgs = ["centos-release", "usermode", "rhn-setup", "python-syspurpose"]
+cos8pkgs = ["centos-linux-release", "usermode", "rhn-setup", "python3-syspurpose"]
+# The packages 'python-syspurpose' and 'python3-syspurpose' were removed in Oracle Linux 7.9
+# and Oracle Linux 8.2 respectively.
+# Installing and checking for these packages (just to make sure)
+# from CentOS repositories to cover all distros consistently.
+ol7repobypass = "http://mirror.centos.org/centos/7/os/x86_64/Packages/python-syspurpose-1.24.42-1.el7.centos.x86_64.rpm"
+ol8repobypass = (
+    "https://vault.centos.org/centos/8/BaseOS/x86_64/os/Packages/python3-syspurpose-1.28.21-3.el8.x86_64.rpm"
+)
+
+
+def define_pkg(shell, osys="", pkgs=None):
+    if osys in plt:
+        for pkg in pkgs:
+            print(f"PREP: Setting up {pkg}")
+            query = shell(f"rpm -q {pkg}")
+            if f"{pkg} is not installed" in query.output:
+                if pkg == "python3-syspurpose" and osys == "oracle-8":
+                    shell(f"yum install -y {ol8repobypass}")
```

Comment:
I would rather put assert here. In case the URL stops working or there is some error we need to know. If the `shell` command returns non-zero return code the pytest won't report a failure.
```suggestion
                    assert shell(f"yum install -y {ol8repobypass}").returncode == 0
```

Reply:
After the disscussion with @kokesak and @mkluson it has been decided to abandon checking for python-syspurpose and python3-syspurpose on Oracle Linux completely as these are not available on convertable versions of OL